`move_as_ptr!` not pointing to correct crate

## Bevy version and features

- 0.17-rc1


## What you did

Import `MovingPtr` and `move_as_ptr` from `bevy::ptr`

## What went wrong


```
error[E0433]: failed to resolve: could not find `MovingPtr` in `bevy_ptr`
  --> src/preset/spatial.rs:74:9
   |
74 |         move_as_ptr!(xy);
   |         ^^^^^^^^^^^^^^^^ could not find `MovingPtr` in `bevy_ptr`
   |
   = note: this error originates in the macro `move_as_ptr` (in Nightly builds, run with -Z macro-backtrace for more info)
help: a struct with a similar name exists
  --> /home/lenny/.cargo/registry/src/index.crates.io-1949cf8c6b5b557f/bevy_ptr-0.17.0-rc.1/src/lib.rs:1211:41
   |
121-         let $value = unsafe { bevy_ptr::MovingPtr::from_value(&mut $value) };
121+         let $value = unsafe { bevy_ptr::OwningPtr::from_value(&mut $value) };
   |
help: consider importing this struct through its public re-export
   |
 1 + use crate::preset::spatial::MovingPtr;
```

--- a/crates/bevy_ptr/src/lib.rs
+++ b/crates/bevy_ptr/src/lib.rs
@@ -1212,7 +1212,7 @@ macro_rules! move_as_ptr {
         //   it is impossible to refer to the original value, preventing further access after
         //   the `MovingPtr` has been used. `MaybeUninit` also prevents the compiler from
         //   dropping the original value.
-        let $value = unsafe { bevy_ptr::MovingPtr::from_value(&mut $value) };
+        let $value = unsafe { $crate::MovingPtr::from_value(&mut $value) };
     };
 }
 

